libs/ui/src/lib/fire-calculator/fire-calculator.component.html

```diff
@@ -17,24 +17,36 @@
           <span class="ml-2" matTextSuffix>{{ currency }}</span>
         </mat-form-field>
 
+        <mat-form-field appearance="outline" class="w-100">
+          <mat-label i18n>Annual Interest Rate</mat-label>
+          <input
+            formControlName="annualInterestRate"
+            matInput
+            step="0.25"
+            type="number"
+          />
+          <div class="ml-2" matTextSuffix>%</div>
+        </mat-form-field>
+
         <mat-form-field appearance="outline" class="w-100">
           <mat-label i18n>Investment Horizon</mat-label>
           <input formControlName="time" matInput type="number" />
           <span class="ml-2" i18n matTextSuffix>years</span>
         </mat-form-field>
 
         <mat-form-field appearance="outline" class="w-100">
-          <mat-label i18n>Annual Interest Rate</mat-label>
+          <mat-label i18n>Target Net Worth</mat-label>
           <input
-            formControlName="annualInterestRate"
+            formControlName="retirementNetWorth"
             matInput
-            step="0.25"
+            step="100"
             type="number"
           />
-          <div class="ml-2" matTextSuffix>%</div>
+          <span class="ml-2" matTextSuffix>{{ currency }}</span>
         </mat-form-field>
 
         <gf-value
```

Comment:
Always show the input (not as `gf-value`) that you can directly edit (either _Investment Horizon_ or _Projected Total Amount_)

Reply:
Changed Projected Total amount to an input field.